webapp/src/utils/index.ts

```diff
@@ -44,19 +44,10 @@ export const getProjectLinkModalArgs = (str: string): LinkPayload => {
     };
 };
 
-export const getCreateTaskModalCommandArgs = (arr: Array<string>): CreateTaskCommandArgs => {
-    if (arr.length < 3) {
-        return {
-            title: '',
-            description: '',
-        };
-    }
-
-    return {
-        title: arr[1] ?? '',
-        description: arr[2] ?? '',
-    };
-};
+export const getCreateTaskModalCommandArgs = (arr: Array<string>): CreateTaskCommandArgs => ({
+    title: arr[1] ?? '',
+    description: arr[2] ?? '',
```

Comment:
What if array has a length less than 3?

Reply:
in that case `arr[1], arr[2]` will be `undefined` and we will have empty string

Reply:
Are you sure you won't get error for accessing indices of an array more than the length?

Reply:
Yes, in JS it will not throw any error for this